x-pack/plugins/infra/public/pages/metrics/layouts/nginx.ts

```diff
@@ -20,7 +21,12 @@ export const nginxLayoutCreator: InfraMetricLayoutCreator = theme => [
     sections: [
```

Comment:
Shouldn't the "Nginx" label above be translated under the `xpack.infra.metricDetailPage.nginxMetricsLayout.layoutLabel` id?

Reply:
According to [Glossary](https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit) we shouldn't translate this label.